docs/public/deploying-java-app-with-database.adoc

```diff
@@ -44,53 +44,13 @@ $ kubectl create -f https://operatorhub.io/install/postgresql-operator-dev4devs-
 
 **NOTE**: The `my-postgresql-operator-dev4devs-com` Operator will be installed in the `my-postgresql-operator-dev4devs-com` namespace and will be usable from this namespace only.
 
-==== Creating a database to be used by the sample application
```

Comment:
I moved this section after `odo create` as it is now (from a previous PR) necessary to have a component before to create a service.